target/linux/ipq806x/base-files/etc/board.d/02_network

```diff
@@ -22,6 +22,10 @@ tplink,vr2600v)
 	ucidef_add_switch "switch0" \
 		"1:lan" "2:lan" "3:lan" "4:lan" "6@eth1" "5:wan" "0@eth0"
 	;;
+askey,rt4230w)
```

Comment:
askey comes before buffalo ...

Reply:
Despite, this can be merged with nec,wg2600hp; so create one case entry for both of them and move it up.